src/smc-webapp/projects.cjsx

```diff
@@ -697,10 +698,11 @@ NewProjectCreator = rclass
 
     cancel_editing: ->
         @setState
-            state            : 'view'
-            title_text       : ''
-            description_text : ''
-            error            : ''
+            state             : 'view'
+            title_text        : ''
+            description_text  : ''
+            error             : ''
+            create_button_hit : ''
```

Comment:
Comment what the possible states are. Not a huge fan of this name but it makes okay sense knowing what the possible states are.